functions/index.js

```diff
@@ -160,12 +195,27 @@ exports.startNewSprint = functions.https.onRequest((request, response) => {
             .then(function(doc) {
                 newSprintId = doc.data().CurrentSprintId + 1;
                 var newSprintIdString = "S" + newSprintId.toString();
-
+                if (doc.exists)
+                {
                 var setNewSprintPromise = db.collection("Main").doc(newSprintIdString).update({
+                    TotalDevelopmen: totalDevelopment,
+                    TotalBusiness: totalBusiness,
+                    TotalMarketing: totalMarketing,
                     EndDate: endDate,
                     StartDate: startDate,
                     Status: status
-                });
+                });}
+                else
+                {
+                    var setNewSprintPromise = db.collection("Main").doc(newSprintIdString).set({
```

Comment:
var setNewSprintPromise is already used in line no. 200
change this variable name

Reply:
sure